ShadowDOM/explainer.md

```diff
@@ -262,12 +263,28 @@ A global map does come with some tradeoffs, particularly when names collide. Wit
+4. HTML documents are already using fragments for many different concepts, such as [fragment navigations](https://html.spec.whatwg.org/#navigate-fragid), [history updates](https://html.spec.whatwg.org/#url-and-history-update-steps), [internal resource links](https://html.spec.whatwg.org/#process-internal-resource-links), [SVG href targets](https://www.w3.org/TR/SVG2/struct.html#UseElement), and more. Although these use cases are very different, a common factor between them is that they all reference elements in the main document, and cannot refer to elements within a shadow root. An important piece of this proposal is that nested shadow roots can modify the global module map. Introducing a new scoping behavior for fragments that does not fit this model could be confusing to authors.
+5. URL's that consist only of a fragment resolve to a [relative URL](https://url.spec.whatwg.org/#relative-url-string), with the base url defined as the source document per [the URL parsing algorithm](https://url.spec.whatwg.org/#url-parsing). This means that using a fragment-only syntax (which would be desired in this scenario) could break if a [`<base>` element](https://html.spec.whatwg.org/#the-base-element) exists that remaps the document's base URL.
+
+Another alternative could be to define a new [scheme](https://url.spec.whatwg.org/#concept-url-scheme) for local references. This is a potential solution, however, since the containing HTML document already has a scheme, this option would require developers to always specify the [absolute URL with fragment string](https://url.spec.whatwg.org/#absolute-url-with-fragment-string) for local specifiers rather than just the fragment (a fragment-only URL is valid due to the way [relative URL](https://url.spec.whatwg.org/#relative-url-string) processing applies). This could be cumbersome for developers to specifiy a full URL for local references when they only need to provide an identifier. A new scheme could also imply scoping behaviors that are not supported, such as [external-file references](https://www.w3.org/TR/SVG2/linking.html#definitions) that are valid in SVG. It also may not be compatible with existing [custom scheme handlers](https://html.spec.whatwg.org/#custom-handlers).
```

Comment:
Might be worth tossing in the (bad) idea of having the new scheme not be actually tied to an address at all, and really just be a syntax choice. So it could end up being something like `localid://foo`, even though the site is at `https://my.example.site.com`.

This would avoid the problem of having to use the absolute URLs, though it would further increase the potential confusion to the developers.